backend/lucene/src/main/java/org/hibernate/search/backend/lucene/logging/impl/Log.java

```diff
@@ -636,4 +636,8 @@ SearchException cannotUseQueryElementForObjectFieldBecauseCreationException(Stri
 			value = "Unable to create instance of analysis component '%1$s': %2$s")
 	SearchException unableToCreateAnalysisComponent(@FormatWith(ClassFormatter.class) Class<?> type, String causeMessage,
 			@Cause Exception cause);
+
+	@Message(id = ID_OFFSET + 143,
+			value = "Error while applying query cache configuration: %1$s")
```

Comment:
To be consistent with `unableToApplyAnalysisConfiguration()`:

```suggestion
			value = "Unable to apply query caching configuration: %1$s")
```

Reply:
I'll change it.